monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js

```diff
@@ -146,10 +146,6 @@ class ReportPageComponent extends AuthComponent {
         [this.issueContentTypes.REPORT]: strongUsersOnCritIssueReport,
         [this.issueContentTypes.TYPE]: this.issueTypes.DANGER
       },
-      'weak_password': {
-        [this.issueContentTypes.OVERVIEW]: weakPasswordIssueOverview,
-        [this.issueContentTypes.TYPE]: this.issueTypes.DANGER
-      },
```

Comment:
If I'm not wrong, this is the issue that tells the user that configured credentials were used successfully? Why are we removing this?

Reply:
We don't have any way of telling that a configured credentials was used successfully on the frontend. This will be added later. I will add a note on it.